libzhlgen/test/functions/win32/1.6.9/freetype.zhl

```diff
@@ -15,6 +15,17 @@ static cleanup uint64_t freetype::easy_printCenter_DO_NOT_USE_DIRECTLY(int fontS
 ".8d4c240483e4f0ff71fc5589e55756535181ec????????8b5110":
 static cleanup __cdecl void freetype::easy_printAutoShrink(int fontId, float x, float y, int maxWidth, bool centered, const std::string& text);
 
+"55660fefc9660fefc089e55389cb83e4f083ec20":
+__thiscall void freetype::font_data::init(freetype::font_data *this, const void *buffer, int bufferSize, unsigned int h, bool glow);
+"5589e583e4f083ec108b4104890424e8????????c9c3":
+__thiscall void freetype::font_data::clean(freetype::font_data *this);
+"5589e5565383e4f083ec308b4d08f30f104d0c85c97e??3b0d????????7d??803d????????008b15????????74??8b1d????????39d97c??8d349d000000008b143285d274??89c829d88db60000000039c389c17f??8b143229d885d275??660fefc0eb??8b1c8a660fefc085db74??0f2ec173??0f2ec876??0f2e4b247a??75??f30f10432c":
```

Comment:
`font_height` must be declared ***before*** `font_baseline` in Linux
`font_height` should be declared ***after*** `freetype::easy_printAutoShrink`

`font_height` Linux 64: `.85ff530f8e`
`font_height` Linux 32: `.57565383ec208b4c2430f30f1044243485c9`
`font_baseline` Linux 64: `.85ff530f8e`
`font_baseline` Linux 32: `.57565383ec208b4c2430f30f1044243485c9`